internal/pkg/app/handlers_v1.go

```diff
@@ -679,6 +734,10 @@ func convertToApiError(ctx context.Context, err error) ApiError {
 		return &BadRequestError{"bad route config", err}
 	} else if errors.As(err, &routeNotFound) {
 		return &NotFoundError{"route " + routeNotFound.RouteId + " not found"}
+	} else if errors.As(err, &jwtAuthError) {
```

Comment:
We should move these errors to middlewares.go as well

Reply:
it's convenient to keep these there (it's the same package anyways) unless there is a strong reason not to